ioctl/client.go

```diff
@@ -102,16 +102,9 @@ func (c *client) Config() config.Config {
 }
 
 func (c *client) AskToConfirm() bool {
-	msg, lang := c.SelectTranslation(confirmMessages)
-	fmt.Println(msg)
```

Comment:
if there is always a message before user input, we can keep the `msg` as an input parameter of this function.

Reply:
yes,  i  search  ioctl/cmd.  i can keep msg as the parameter